docs/DeveloperGuide.adoc

```diff
@@ -782,13 +782,16 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* tech-savvy individual
+* can type relatively fast while maintaining accuracy
+* prefers being on desktop rather than on mobile
+* generally comfortable with typing on CLI applications without clicking
```

Comment:
Maybe instead of writing "generally comfortable with typing...", it will be better to show that they need such speed and efficiency when dealing with large numbers of animals daily (and especially more so in the cases of emergency). So it's more of a necessity for them to be able to just type and search for records fast without even reaching for the mouse because they do not want to waste time finding buttons and clicking through the application. Maybe rephrase to something like "Requires a CLI application due to <reason>".